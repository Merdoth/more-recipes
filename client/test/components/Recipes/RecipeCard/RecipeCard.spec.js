import React from 'react';
import { shallow } from 'enzyme';
import expect from 'expect';
import { RecipeCard } from
  '../../../../components/Recipes/RecipeCard/RecipeCard.jsx';

const setup = () => {
  const props = {
    recipeList: {},
  };
  return shallow(<RecipeCard {...props} />);
};

describe('RecipeCard Component snapshot', () => {
  it('it should render the right amount of elements', () => {
    const wrapper = setup();
    expect(wrapper).toMatchSnapshot();
  });
});
