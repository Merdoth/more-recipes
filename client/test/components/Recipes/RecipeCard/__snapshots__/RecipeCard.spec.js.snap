// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RecipeCard Component snapshot it should render the right amount of elements 1`] = `
<div
  className="col-md-4 col-sm-4 main-card"
>
  <div
    className="top-items"
  >
    <Link
      className="rated"
      replace={false}
      to="recipe-details/undefined"
    >
      <RecipeCardImage />
      <RecipeCardDes />
      <RecipeCardFooter />
    </Link>
  </div>
</div>
`;
